recipes/djinni-support-lib/1.x.x/conanfile.py

```diff
@@ -0,0 +1,128 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class DjinniSuppotLib(ConanFile):
+    name = "djinni-support-lib"
+    homepage = "https://djinni.xlcpp.dev"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Djinni is a tool for generating cross-language type declarations and interface bindings"
+    topics = ("java", "Objective-C", "Android", "iOS")
+    license = "Apache-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+                "fPIC": [True, False],
+                "target": ["jni", "objc", "python", "cppcli", "auto"],
+                "system_java": [True, False]
+               }
+    default_options = {"shared": False,
+                        "fPIC": True ,
+                        "target": "auto",
+                        "system_java": False,
+                       }
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def objc_support(self):
```

Comment:
```suggestion
    def _objc_support(self):
```

Reply:
Should I update the [old conanfile](https://github.com/conan-io/conan-center-index/blob/master/recipes/djinni-support-lib/all/conanfile.py#L29) as well, while we're at it? Or should I keep it unchanged or fix it in a separate PR?